src/System.Net.Security/src/System/Net/CertInterface.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using System.Net.Security;
+using System.Security.Cryptography.X509Certificates;
+
+namespace System.Net
+{
+    internal abstract class CertInterface
```

Comment:
NIT: Is this part of the PAL? Please rename to get the code uniform - see Primitives, NameResolution and Sockets for a model.
I would recommend `CertInterfacePAL` instead.